tools/docker/syzbot/provision-llvm.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+# Copyright 2024 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+set -e # exit on any problem
```

Comment:
Please put the comment on a separate line, start it with a capital letter and end with a period.